Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/RulesetManagement.java

```diff
@@ -321,5 +326,50 @@ public boolean isAlwaysShowingForKey(String keyId) {
         return false;
     }
 
+    @Override
+    public int updateMetadata(Collection<Metadata> currentMetadata, String acquisitionStage,
+            Collection<Metadata> updateMetadata) {
+
+        final Function<String, MutableTriple<Collection<Metadata>, Reimport, Collection<Metadata>>> entryProducer = key -> {
+            var entry = new MutableTriple<Collection<Metadata>, Reimport, Collection<Metadata>>();
```

Comment:
Here `var` abbreviates `MutableTriple<Collection<Metadata>, Reimport, Collection<Metadata>>`, therefore the use seems appropriate to me. @solth is this okay for you?

Reply:
I do think we should be careful and do not become "overzealous" when making use of the `var` type inference, but it would probably be acceptable in this particluar case because the type is right there on the right hand side of the variable assignement. 

On the other hand, this is perhaps a topic that should be discussed more thoroughly and the result of the discussion should be included in the coding guidelines. Some seem to consider it a "code smell" (https://medium.com/@spextreme/java-var-should-be-a-code-smell-17af0be79646) while others think the advantages outweigh the risks (https://www.beyondjava.net/should-you-embrace-var-in-java-10)

So perhaps the best solution would be to stick to explicit variable typing for now and only use `var` once we have come to a general decision concerning its usage.